grassroots-backend-e2etests/src/PhoneCanvass.e2e-spec.ts

```diff
@@ -12,6 +13,60 @@ import { UsersModule } from "grassroots-backend/users/Users.module";
 import { TwilioService } from "grassroots-backend/phone-canvass/Twilio.service";
 import { TwilioServiceMock } from "grassroots-backend/phone-canvass/Twilio.service.mock";
+import {
+  CallStatus,
+  TwilioCallStatus,
+} from "grassroots-shared/dtos/PhoneCanvass/CallStatus.dto";
+
+function getFormDataForCSVCreation(): FormData {
+  // Create a mock audio file
+  const audioBuffer = Buffer.from("fake audio content");
+  const audioFile = new File([audioBuffer], "test-voicemail-audio.mp3", {
+    type: "audio/mpeg",
+  });
+
+  // Create FormData with CSV and audio
+  const formData = new FormData();
+  formData.append("name", "test");
+  formData.append(
+    "csv",
+    `id,civi_id,voter_id,seq_id,district_num,district,poll,first_name,middle_name,last_name,language_pref,unit_num,bldg_num,bldg_num_sfx,street_name,street_type,street_dir,address,town,postal_code,province,phone,do_not_phone,do_not_mail,do_not_email,support_level,party_support,volunteer_status,volunteer_tasks,volunteer_notes,description,membership_status,membership_join_date,membership_expiry_date,voted,election_voted_in,tags,email,merge_tag_token
+sbcw8nlvwwwcqc95jsf35103,1,1,,1,Kitchener Centre,400,Alex,Aaron,Aardvark,en,,,,Astreet,St,,,Kitchener,N2G 8A7,ON,2267382384,,,,1,GPO,pending,"data_entry, driver, foot_canvassing, mainstreeting, phoning, scrutineering, signs",,,Lapsed (Non-Voting),2024-03-18,2025-09-01,,,;;import-2023-07-24; 2022-gpc-donor; true-multi-unit,a@a.com,
+wvo53djjer4hfn7fpazvkjtu,2,2,,2,Kitchener Centre,400,Bob,Billy,Burnham,en,,,,Bstreet,St,,,Kitchener,N2G 8A8,ON,2267382385,,,,2,GPO,pending,"data_entry, driver",Nov '24 - GPO staff (tech),Nov '24 - GPO staff (tech),,,,,,,b@b.com,
+e2abnzfhayfnlkc8galofnz5,3,3,,3,Kitchener Centre,400,Cassy,Casey,Clever,en,1,1,,Cstreet,Rd,,,Leamington,N8H 2Q8,ON,5147382927,,,,3,,,,Som great notes.,,,,,,,; ; vlu-43-02-ole-2022; fed-2021-sg; pre-2023-sg; plu-fed-2023; 2021-gpc-donor; donor-400+; import-2023-07-24; 2022-gpc-donor; true-multi-unit; import-2023-08-24; fed-2021-voted; 413-greenbelt; import-2023-10-24; active2-oct25; vlu-43c-01-ple-2023; ,c@c.com,
+2daxkrhawl0vlez883m9s17m,4,4,,4,Kitchener Centre,400,Durham,Dally,Dietrich,en,2,1,,Dstreet,Rd,,,Leamington,N8H 2Q9,ON,5147382928,,,,5,,,phoning,,,,,,,,; ; vlu-43-02-ole-2022; fed-2021-sg; pre-2023-sg; plu-fed-2023; 2021-gpc-donor; donor-400+; import-2023-07-24; 2022-gpc-donor; true-multi-unit; import-2023-08-24; fed-2021-voted; 413-greenbelt; import-2023-10-24; active2-oct25; vlu-43c-01-ple-2023; volunteer-prospect; vote-pledge-2023; vlu-43c-03-pdl-2023; ac23-eday-vol-confirmed; ac23-wat-reg-vol; 2024-legalizeit-postcard; 2024-postelxn-textblast; pre-2024-sign-request; pre-2024-active3-vol; vlu-43-03-au-2024; ; ; 2023-advancepoll-voter; ac25voteadvance1; eday-vol-confirmed-2025; ; ; ; ; ; ; ;,d@d.com,`,
+  );
+  formData.append("voiceMailAudioFile", audioFile);
+  return formData;
+}
+
+async function updateTwilioCallStatus(
```

Comment:
As we build out more tests, I think we'll need a bunch of helpers like this to create a little DSL type thing.